src/components/templates/search-modal/index.tsx

```diff
@@ -102,7 +102,7 @@ const SearchModal = ({ handleClose }) => {
                   onClick={handleClear}
                   content="Clear search"
                 >
-                  <CrossIcon className="flex text-grey-50" />
```

Comment:
Hi @aliseidev, 

Thanks for the PR, and sorry for taking so long to review it. If you could please remove this onClick you've added. as the button is not supposed to close the search modal, but just clear the text field. Otherwise, it looks great 😄